templates/demo/header-template.tsx

```diff
@@ -0,0 +1,10 @@
+import { Title } from '@patternfly/react-core';
```

Comment:
Do we want to have the `Demo` template for some time? I would say header and footer should be almost the same on all templates (if someone would want to use custom they would have to provide it). So we can by all means rename this from `Demo` to `Regular` or something like that.

Reply:
Yes, that is a plan to have common headers and footers. I will create a new task for that.

Reply:
Okay, once we do that let's remove the `Demo` templates. This way, we'll somewhat force other contributors to automatically opt-in the default one and we won't end with multiple variants of the same header and footer.

Reply:
I mostly created the demo template for local testing cause the aa templates require data API and we don't have that setup for local development just yet.